Android/app/src/main/java/io/github/project_travel_mate/notifications/NotificationsActivity.java

```diff
@@ -70,7 +73,12 @@ protected void onCreate(Bundle savedInstanceState) {
         notifications = new ArrayList<>();
 
         getNotifications();
-
+        mySwipeRefreshLayout.setOnRefreshListener( new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                getNotifications();
```

Comment:
show the animation by the time API is being called.

Reply:
Shall I remove the default circular progress bar which we get ?

Reply:
Keep the animation.

If you feel like the circular progress bar doesn't look good above the animation, you can remove that.